docs/advanced/navigating-politeia-data.md

```diff
@@ -2,101 +2,83 @@
+| -------------------- | ---------------------- |
+| **/payload**           | Folder containing an `index.md file`, which has the full text of the proposal and any images associated with the proposal.    |  
+| **/plugins/decred**         |  Folder containing the `comments.journal` file (which contains comments and up/down vote data) and, for proposals that have opened for voting, a `ballot.journal` file, which contains ticket voting data.   |  
+| **00.metadata.txt**          | Data about the proposal submission:   <ul style="font-size: 13px"><li>timestamp</li><li>pubkey of submitter</li><li>title</li> <li>signature</li></ul> |  
+| **02.metadata.txt**           | Data about admin approval for display: <ul style="font-size: 13px"><li>timestamp</li><li>pubkey of admin</li></ul>     |  
+| **13.metadata.txt:**           | Data about proposal owner authorizing the start of voting. Where proposal owner has authorized and then rescinded authorization this will appear in multiple commits:  <ul style="font-size: 13px"><li>timestamp</li><li>owner pubkey </li><li>transaction hash</li></ul>   |  
+| **14.metadata.txt**          | Data about admin authorizing the start of voting. Includes specification of the vote: duration (2016 blocks):   <ul style="font-size: 13px"><li>quorum requirement (20%)</li><li>approval threshold (60%)</li><li>id and descriptions for voting options</li></ul> |  
+| **15.metadata.txt**          | Data about voting period:  <ul style="font-size: 13px"><li>starting block height and hash</li><li>ending block height </li><li>a list of every ticket eligible to vote on the proposal</li></ul>    |  
+| **recordmetadata.json**          | Metadata about the record.   |  
 
-**recordmetadata.json**: 
 
-The plugins/decred files warrant further explanation.
+## Voting and comments data
 
-The ballot and comment journal files are updated on the server in real time as votes are cast and comments are made. Every hour, a git commit is made for the latest journal file changes. Making a git commit is expensive performance wise so making a new commit every time a comment was made or a vote was cast would not be practical.
+Data on votes cast by ticket holders on a given proposal is stored in the `ballot.journal` file in the `/plugins/decred` folder. Comment data (comment content and up/down votes on comments) is stored in the `comments.journal` file in the same folder. These files are updated on the server in real-time as votes are cast and comments are made. Every hour, a git commit is made updating the `comments.journal` and (if voting has started) `ballot.journal` file ---versions of the proposal submitted prior to the version that is voted on will only have a `comments.journal` file. Commits are made every hour because making a git commit is expensive performance-wise, and making a commit for every vote and comment would not be practical. Additionally, grouping votes in hourly commits helps protect the privacy of ticketholders.
```

Comment:
> Every hour, a git commit is made updating the `comments.journal` and (if voting has started) `ballot.journal` file ---versions of the proposal submitted prior to the version that is voted on will only have a `comments.journal` file.

I had to re-read the sentance quoted above a couple of times. How about simplifying to something like

> A git commit is made every hour to update the `comments.journal` file. If voting has started on a proposal, the same commit will also be used to update the `ballot.journal` file.

Reply:
This comment made me rethink the structure of the subheadings generally. To make the document more consistent, have created three new subheadings:

### Voting and comment data
#### Vote data
#### Comment data
### User data

I still think we should have separate tables for `ballot.journal` and `comments.journal`, as they do have their own internal sets of parameters. Have put these in their respective 'vote data' and 'comment data' subheadings, where they seem to make more sense.

Reply:
I like the new structure & headings, but I'm still not convinced by the paragraph I highlighted above. What do you think of this rewording:

> A git commit is made every hour for each active proposal to update the `comments.journal` file. If voting has started on a proposal, the same commit will also be used to update its `ballot.journal` file. The hourly commits are stopped once the voting has been completed and all votes have been recorded in git. Commits are made every hour because making a git commit is expensive performance-wise and making a commit for every vote and comment would not be practical. Additionally, grouping votes in hourly commits helps protect the privacy of ticketholders.

Reply:
More concise and clear. Used rewording.